EvaluationModule/src/deepeval_engine/deepeval_evaluator.py

```
@@ -357,81 +466,269 @@ def evaluate_test_cases(
+                                        actual_output=turn_content,
+                                        expected_output=""
+                                    )
+                                    toxicity_metric = ToxicityMetric(
+                                        threshold=self.metric_thresholds.get("toxicity", 0.5),
+                                        model=judge_llm if judge_provider == "openai" else None
+                                    )
+                                    try:
+                                        toxicity_metric.measure(turn_test_case)
+                                        if toxicity_metric.score is not None:
+                                            toxicity_scores.append(toxicity_metric.score)
+                                    except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)